SystemSprawozdan/SystemSprawozdan.Frontend/Shared/UI/UserPad.razor

```diff
@@ -21,9 +21,19 @@
 </div>
 
 @code {
+	/// SPRAWDZIĆ!
 	public string FirstName { get; set; }
 	public string LastName { get; set; }
 
+	public UserInfoGetDto User = new();
+	protected override async Task OnInitializedAsync()
+	{
+		/// Wiem, pamiętam! Poprawię jak będę pushować Teachera ;)
+		//User = await AppHttpClient.Get<UserInfoGetDto>("Account/userCredentials");
```

Comment:
jak przypisujesz wynik zapytania do pola User to mozesz je wkorzystac do wyswietlenia imienia i nazwiska zamiast ponownie przepiswac je do kolejnych pól